### PR TITLE
Feature/login register services

### PR DESCRIPTION
**Pull request by Theepanguna**
Based on the advice on Login service, I did the register service.

### CODE REVIEW

File: src/app/services/login.service.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+  // login(email: string, password: string): Observable<any> {
+  //   const header= new HttpHeaders({
+  //     'Content-Type': 'application/json',
+  //   })
+  //   const url = `${this.tournamentUrl}/login`;
+  //   const body = { email, password };
+  //   return this.http.post(url, body, { headers: header });
+  // }
```
Comment (SebastianSilvaDev) [inline src/app/services/login.service.ts]: If not needed remove this comment

**Review by SebastianSilvaDev — CHANGES_REQUESTED** [review]
Confirm if you already tested it, with and it returns the appropieate responses for the requests.

Also, do not create another PR, just comment on the other one

File: src/app/services/login.service.ts (hunk 1 above)
Comment (Theepanguna) [reply src/app/services/login.service.ts]: sure

**Comment by Theepanguna** [comment]
Adjusted the Login parsing username and password, but getting error when using matdialogref for register

**Comment by Theepanguna** [comment]
@respatica hi, did some login and register form control, can you complete the balance or guide.

**Comment by Theepanguna** [comment]
I have added error notification to the user when login failed. Added password and confirm password checking and prompting  dialog for user

**Comment by Theepanguna** [comment]
HI , Could you please review this at your free time. cheers!

**Review by SebastianSilvaDev — APPROVED** [review]
LGTM

**Pull request merged**